packages/suite/src/components/wallet/TransactionItem/components/TransactionHeading/index.tsx

```diff
@@ -14,7 +15,10 @@ const Wrapper = styled.span`
     cursor: pointer;
 `;
 
-const HeadingWrapper = styled.div``;
+const HeadingWrapper = styled.div`
+    text-overflow: ellipsis;
+    overflow: hidden;
```

Comment:
Beware of using `overflow: hidden`, it could break the blurs in "discreet mode".

Not caused by this exact line, but see the difference between this branch and production:
<img width="641" alt="image" src="https://user-images.githubusercontent.com/3729633/154992235-4ac133ac-4642-43a7-a396-84802911a07b.png">

<img width="544" alt="image" src="https://user-images.githubusercontent.com/3729633/154992246-49ef177e-5ca8-4901-8884-469365daf47e.png">

Reply:
fixed in https://github.com/trezor/trezor-suite/pull/4144/commits/014db6b9e6eff7541044f0af56a59237b645a338

Reply:
the solution with negative margins is hard to manage, it's usually possible to get rid off the overflow:hidden on those places, but I know it's already there on some other places so ok for now...